skeletons/src/main/java/com/decathlon/vitamin/compose/skeletons/SkeletonColors.kt

```diff
@@ -0,0 +1,10 @@
+package com.decathlon.vitamin.compose.skeletons
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.graphics.Color
+
+@Immutable
+class SkeletonColors(
+    val background: Color,
```

Comment:
In all modules, we have `backgroundColor`. Probably a good idea to rename it and maybe add this rule in the `CONVENTIONS.md` file in this PR or in a next contribution?

Reply:
good idea

Reply:
You've updated the documentation but not the code here x)